doc/src/devdocs/probes.md

```diff
@@ -137,9 +137,6 @@ fib(x) = x <= 1 ? 1 : fib(x-1) + fib(x-2)
 beaver = @spawn begin
     while true
         fib(30)
-        # This safepoint is necessary until #41616, since otherwise this
-        # loop will never yield to GC.
-        GC.safepoint()
```

Comment:
Could we make it conditional `@static if VERSION>=` so that someone on an older Julia versions doesn't hang their process?